vendor/github.com/openshift/api/console/v1/zz_generated.featuregated-crd-manifests.yaml

```diff
@@ -0,0 +1,225 @@
+consoleclidownloads.console.openshift.io:
+  Annotations:
+    description: Extension for configuring openshift web console command line interface
```

Comment:
You capitalized OpenShift further down. Should this one be capitalized too?

Reply:
Looks like some of the descriptions in this file capitalize and some don't. I am thinking they should all be capitalized.

Reply:
@opayne1 thanks for the comment. These changes are vendored from the openshift/api repo, where we need to do the change. I suggest to go though all the console related API in that repo and fix it there. After that will vendor it back to both console and console-operator repo.